projects/novo-elements/src/elements/tooltip/Tooltip.directive.ts

```diff
@@ -87,7 +90,18 @@ export class TooltipDirective implements OnDestroy, OnInit {
     }
   }
 
+  ngAfterViewInit(): void {
+    if (this.overflow && this.elementRef?.nativeElement) {
+      this._resizeObserver = new ResizeObserver(() => {
+        const isOverflowing = this.elementRef.nativeElement.scrollWidth > this.elementRef.nativeElement.clientWidth;
```

Comment:
Do you run into any issues with this around change detection where it doesn't re-render based on `active`? You might need to wrap the callback code within: `this.zone.run(() =>`

https://dev.to/christiankohler/how-to-use-resizeobserver-with-angular-9l5